types/shelljs/index.d.ts

```diff
@@ -965,6 +966,51 @@ export type ShellString = string & ShellReturnValue;
 
 export type ShellArray = string[] & ShellReturnValue;
 
+export const ShellString: ShellStringFunction;
```

Comment:
I&nbsp;prefer putting the&nbsp;<code>export&nbsp;const</code> declaration at&nbsp;the&nbsp;end:
```suggestion
```